frappe/website/doctype/personal_data_deletion_request/personal_data_deletion_request.js

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) 2019, Frappe Technologies and contributors
+// For license information, please see license.txt
+
+frappe.ui.form.on('Personal Data Deletion Request', {
+	refresh: function(frm) {
+		if(frappe.user.has_role('System Manager') && frm.doc.status == 'Pending Approval'){
+			frm.add_custom_button(__('Delete Data'), function() {
+				return frappe.call({
```

Comment:
frappe.call can be written better
```
frappe.call("frappe.method.path", { args_as_json }).then( callback_function )
```

Reply:
this is more clean and makes it easy to read